### DISCUSSION TITLE
[Question][Hw3]

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#522] [Q&A]
Is it allowed to add "open Llvm" to the top of the instrument.ml file?

**Comment by Junseon00** [comment]
Do you want to use `Llvm`?
Then you just can use `Llvm` without open it.

**Comment by KihongHeo** [comment]
If you want to avoid repeating `Llvm.X`, then it is ok to open it.
It is allowed.

**Comment by ee12ha0220** [comment]
Oh I see, I was intending to avoid repeating Llvm.X. Thank you :)

**Comment by mrpositron** [comment]
Could you give a title to your posts? Something like this:
`[Question] [Hw3] ...description of your problem ...`

This will greatly help other people to navigate through the issues board.
